dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/Subsection.test.tsx

```diff
@@ -28,6 +28,7 @@ const defaultProps = {
         title: "Enable External Database",
         type: "boolean",
         value: true,
+        disable: "externalDatabase",
```

Comment:
It'd be a tiny bit clearer if this was `disables: "externalDatabase"` - as in, this param disables the `externalDatabase` param (`disable` without the `s` could mean disable this current param, or disable the following param).